platform/podjs/bin/genPodjs.js

```diff
@@ -0,0 +1,75 @@
+const MagicString = require("magic-string");
+const fs = require("fs");
+
+function replaceManifestData(code, manifestData) {
+    const magicString = new MagicString(code);
+    const pattern = new RegExp("window.manifestData", "g");
+
+    let hasReplacements = false;
+    let match;
+    let start;
+    let end;
+    let replacement;
+
+    while ((match = pattern.exec(code))) {
+        hasReplacements = true;
+
+        start = match.index;
+        end = start + match[0].length;
+
+        replacement = String(manifestData);
+
+        magicString.overwrite(start, end, replacement);
+    }
+
+    if (!hasReplacements) {
+        return null;
+    }
+
+    return magicString.toString();
+}
+
+function executeReplacement(podJs, manifestJsonPath) {
+    try {
+        const manifestData = fs.readFileSync(manifestJsonPath, "utf8");
+        const podJsCode = fs.readFileSync(podJs, "utf8");
+
+        const replacedCode = replaceManifestData(podJsCode, manifestData);
+        if (!replacedCode) {
+            console.log("Nothing to replace into ", podJs);
+            return;
+        }
+
+        fs.writeFileSync(podJs, replacedCode);
+    } catch (err) {
+        console.error("Error during replacing code", err);
+    }
+}
+
+function copyPodJs(dest) {
+    if (!fs.existsSync(dest)) {
+        fs.mkdirSync(dest);
+
+        fs.copyFileSync(
+            "node_modules/@polypoly-eu/podjs/dist/pod.js",
+            `${dest}/pod.js`
+        );
+        console.log("Copied pod.js into", dest);
+    }
+}
+
+/**
+ * @param options
+ * @param options.build_dir - path to build_dir where json and pod.js exist
+ */
+module.exports.loadManifest = (options = {}) => {
```

Comment:
As indicated above, this can simply be a default export, so that you don't need to even call a function name.

Reply:
there's an issue for using `export` here, thus I found this workaround...

Reply:
Which issue? I think that if you use it like `module.exports.default = ...` the import can simply be used as a function.

Reply:
The issue is with how ES6 modules are emulated in CommonJS and how we import the module.
The above with default doesn't work on this setup

Reply:
ok found it can work with `exports.default = loadManifest` too  :)